### ISSUE TITLE
`(scheduled today)` is a malformed query (that could be errored on)

### ISSUE THREAD

**Issue by ParetoOptimalDev** [alphapapa/org-ql#420]
### OS/platform

NixOS

### Emacs version and provenance

NixOS 

### Emacs command

type `emacs` and hit enter in terminal

### Org version and provenance

NixOS: org-9.6.21

### org-ql package version and provenance

NixOS: org-ql-20240222.200

### Actions taken

`/tmp/foo.org` has contents:

```
** TODO walk dog
SCHEDULED: <2024-03-24 Sun ++7d>
```

Then execute this elisp:

```emacs-lisp
(org-ql-search '("/tmp/foo.org") '(scheduled today))
```

### Observed results

I see:

```
  TODO walk dog  in 11d 
```

### Expected results

I expect no results.

### Backtrace

```elisp
n/a
```

**Comment by alphapapa** [comment]
`(scheduled today)` is a malformed query.  You should use either `(scheduled 0)` or `(scheduled :on today)`.  Please see the documentation: https://github.com/alphapapa/org-ql#datetime-predicates  As far as I can tell, there is no example like `(scheduled today)`.

**Comment by ParetoOptimalDev** [comment]
Thank you for the quick response. 

My mistake was caused by a mix of an old saved query having this mistake and being biased towards thinking this was similar to the issue about org-ql not supporting date ranges. This made it harder for me to consider my query syntax for this old view could be wrong.

I skimmed the info manual (thanks for including one btw), but I'll try to read it over again with more care next time.
